MODULE.bazel

```diff
@@ -31,6 +31,10 @@ bazel_dep(
     version = "21.7",
     repo_name = "com_google_protobuf",
 )
+bazel_dep(
+    name = "rules_bazel_integration_test",
```

Comment:
dev_dependency = True

Reply:
I purposefully did not add that. The issue is that a `dev_dependency`  in a dependent module will not be loaded. However, if the release archive ships with any references to the `dev_dependency`, queries will fail. [This issue](https://github.com/keith/buildifier-prebuilt/issues/85#issuecomment-1860509919) represents such a case.

Reply:
Ah, I see.

@Wyverald This seems to be a problem with `dev_dependency` in general. Is there any particular recomendation from the bzlmod team on standard practice for dealing with issues like this?